### PR TITLE
Minor changes for better understanding

### PR DESCRIPTION
**Pull request by Aviral190694**
Minor changes for better understanding of how the protocols Comparable… and Equatable are used and what functions lies in which protocol.

Initially I got confused in terms of why protocols are used and which functions are a part of the same. 
After looking at the protocol https://developer.apple.com/documentation/swift/comparable I got the example. Minor changes can make a person understand the code a bit quicker.

### CODE REVIEW

**Comment by harlanhaskins** [comment]
Ah! This is actually a holdover from the Swift 2 days.

In Swift 3, operators in protocols are implemented using the `static func` syntax that you've changed here. Thanks for catching this!

File: Sources/PlayingCard/Suit.swift [hunk 1]
```diff
@@ -14,19 +14,19 @@ public enum Suit: String {
...
+        case (_, _) where lhs == rhs:
+            return false
+        case (.spades, _),
+        (.hearts, .diamonds), (.hearts, .clubs),
+        (.diamonds, .clubs):
+            return false
+        default:
+            return true
+        }
+    }
```
Comment (harlanhaskins) [inline Sources/PlayingCard/Suit.swift]: Nitpick: Can you remove the trailing whitespace here.

**Comment by aciidb0mb3r** [comment]
I messed up a bit while applying @harlanhaskins's nitpick and this PR got closes. New PR here: https://github.com/apple/example-package-playingcard/pull/4

Thanks @Aviral190694 for the PR!

**Comment by Aviral190694** [comment]
Thank you @harlanhaskins and @aciidb0mb3r. I am so happy to have my first PR accepted in Apple Github Repo! ❤️

**Pull request closed without merging**